lib/driver.js

```diff
@@ -281,10 +292,30 @@ class AndroidUiautomator2Driver extends BaseDriver {
     if (!this.opts.systemPort) {
       return;
     }
-    await PORT_ALLOCATION_GUARD.acquire(AndroidUiautomator2Driver.name,
+    await DEVICE_PORT_ALLOCATION_GUARD.acquire(AndroidUiautomator2Driver.name,
       async () => await this.adb.removePortForward(this.opts.systemPort));
   }
 
+  async allocateMjpegServerPort () {
+    if (!this.opts.mjpegServerPort) {
+      return;
+    }
+    if (this.opts.mjpegServerPort < MJPEG_SERVER_PORT_RANGE[0] || this.opts.mjpegServerPort > MJPEG_SERVER_PORT_RANGE[1]) {
```

Comment:
please remove this check. Let the user to decide the port number

Reply:
OK.

Reply:
This will be resolved in the upcoming changes.